subsys/bluetooth/controller/ll_sw/ull_iso_types.h

```diff
@@ -25,6 +25,18 @@
 #define IS_CIS_HANDLE(_handle) 0
 #endif /* CONFIG_BT_CTLR_CONN_ISO */
 
+
+#if defined(CONFIG_BT_CTLR_ADV_ISO)
+#define IS_ADV_HANDLE(_handle) \
+	(((_handle) >= BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE) && \
+	((_handle) <= (BT_CTLR_ADV_ISO_STREAM_HANDLE_BASE + BT_CTLR_ADV_ISO_STREAM_MAX - 1)))
+#else
+#define IS_ADV_HANDLE(_handle) 0
+#endif /* CONFIG_BT_CTLR_ADV_ISO */
+
+/* ISO LL conformance tests require a PDU size of maximum 251 bytes */
+#define ISO_TX_BUFFER_SIZE 251
+
```

Comment:
```suggestion
```

Revert this change and put back `CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE` as LE Read Buffer Size command allows a range of 1 to 0xFFFF for ISO_Data_Packet_Length